classes/file/FileManager.inc.php

```diff
@@ -234,7 +234,6 @@ function readFileFromPath($filePath, $output = false) {
 	function downloadFile($filePath, $mediaType = null, $inline = false, $fileName = null) {
 		$result = null;
 		if (HookRegistry::call('FileManager::downloadFile', array(&$filePath, &$mediaType, &$inline, &$result, &$fileName))) return $result;
-		$postDownloadHookList = array('FileManager::downloadFileFinished', 'UsageEventPlugin::getUsageEvent');
```

Comment:
I removed the code that clears out the registry before downloading a file; the post download hook list was used to selectively restore hooks after downloading had completed. I don't think it's a good optimization so I removed it.